.internal-ci/helm/fog-ingest/templates/fog-ingest-statefulset.yaml

```diff
@@ -106,27 +108,27 @@ spec:
           valueFrom:
             configMapKeyRef:
               name: fog-recovery-postgresql
-              key: postgresql-hostname
+              key: postgres-hostname
```

Comment:
The original intent was to follow the bitnami chart secret/configmap key convention for our pg secret/configmap. 

I accidentally added the "ql" ending, which lead us to adding in both `postgresql-*` and `postgres-*` .

Since we're redoing how we generate the pg secret/config, this fixes that inconsistency.